frb_codegen/src/main.rs

```diff
@@ -19,11 +19,12 @@ mod generator_dart;
 mod generator_rust;
 mod others;
 mod parser;
+mod source_graph;
 mod transformer;
 mod utils;
 
 fn main() {
-    env_logger::Builder::from_env(Env::default().default_filter_or("info")).init();
+    env_logger::Builder::from_env(Env::default().default_filter_or("debug")).init();
```

Comment:
What about keeping the default filter INFO, o/w users will see lots of log. And you can always change the type via env variables when you are debugging

Reply:
Ah, just an oversight. Changing that now.